src/main/resources/db/migration/data/R__9_7_OFFENDER_TRANSACTION_DETAILS.sql

```diff
@@ -1 +1 @@
-INSERT INTO OFFENDER_TRANSACTION_DETAILS(TXN_DETAIL_ID,TXN_ID,TXN_ENTRY_SEQ,CALENDAR_DATE,PAY_TYPE_CODE,EVENT_ID,PAY_AMOUNT,PIECE_WORK,BONUS_PAY) VALUES(1,301826810, 1, TO_DATE('2020-10-10', 'DD-MM-YYYY'),'UNEMPLOYED', -1, 2.0, 0,0)
\ No newline at end of file
+INSERT INTO OFFENDER_TRANSACTION_DETAILS(TXN_DETAIL_ID,TXN_ID,TXN_ENTRY_SEQ,CALENDAR_DATE,PAY_TYPE_CODE,EVENT_ID,PAY_AMOUNT,PIECE_WORK,BONUS_PAY) VALUES(1,301826810, 1, TO_DATE('2020-10-10', 'YYYY-MM-DD'),'UNEMPLOYED', -1, 2.0, 0,0);
```

Comment:
hsqdb was very lenient with dates - just put this record in anyway even though format incorrect